contrib/mesos/pkg/controllermanager/controllermanager.go

```diff
@@ -160,8 +161,8 @@ func (s *CMServer) Run(_ []string) error {
 		routeController.Run(s.NodeSyncPeriod)
 	}
 
-	resourceQuotaController := resourcequotacontroller.NewResourceQuotaController(kubeClient)
-	resourceQuotaController.Run(s.ResourceQuotaSyncPeriod)
+	go resourcequotacontroller.NewResourceQuotaController(
```

Comment:
@smarterclayton - this is consistent with the other controllers now for how they are launched.

see endpoint controller / replication controller here:

https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/controllermanager.go#L267